src/components/ContentRender/contentRender.css

```
@@ -2,9 +2,11 @@
   --content-render-font-family:
     "Charter", "PingFang SC", "Microsoft YaHei", "Source Han Sans SC",
     sans-serif;
+  --content-render-font-size: var(--mdf-slide-font-size, 16px);
+  --content-render-line-height: var(--mdf-slide-line-height, 30px);
   font-family: var(--content-render-font-family);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)